srcpkgs/ardour/template

```diff
@@ -1,22 +1,47 @@
 # Template file for 'ardour'
 pkgname=ardour
-version=6.0
+version=6.3
 revision=1
+wrksrc="Ardour-${version}.0"
 build_style=waf3
 configure_args="--cxx11 --no-phone-home --with-backends=jack,alsa,dummy
- --libjack=weak --optimize --docs --use-external-libs"
-hostmakedepends="python3 git graphviz doxygen pkg-config clang perl"
+ --libjack=weak --optimize --docs --use-external-libs --freedesktop"
+hostmakedepends="python3 git graphviz doxygen pkg-config clang perl
+ itstool gettext tar readline-devel"
 makedepends="boost-devel liblrdf-devel liblo-devel lilv-devel suil-devel
  taglib-devel aubio-devel rubberband-devel alsa-lib-devel libsndfile-devel
  vamp-plugin-sdk-devel fftw-devel jack-devel libsamplerate-devel lv2
  sratom-devel gtkmm2-devel libarchive-devel fluidsynth-devel hidapi-devel
- libltc-devel"
+ libltc-devel glibmm-devel"
 short_desc="Professional-grade digital audio workstation"
-maintainer="Andrea Brancaleoni <abc@pompel.me>"
+maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="http://ardour.org"
-nocross="executes test binaries during configure"
+distfiles="https://community.ardour.org/src/Ardour-${version}.0.tar.bz2"
+checksum=93418636d7d93f66bdf479ed5bf3c6b0cc0198dc8f76c5555110d3147e4ec74b
 
-do_fetch() {
-	git clone --depth 1 --branch=$version https://github.com/Ardour/ardour $pkgname-$version
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*)  configure_args+=" --dist-target=x86_64" ;;
+	i686*)    configure_args+=" --dist-target=i686" ;;
+	aarch64*) configure_args+=" --dist-target=aarch64" ;;
+	arm*)     broken="glibmm/threads.h not found, wth is going on?" ;;
+	# Some armv7 doesn't have NEON
+	armv7*)   configure_args+=" --dist-target=armhf --no-fpu-optimization" ;;
```

Comment:
Looks unreachable.

Reply:
Yes, intentionally left those configuration until I figure out what's the problem with armv6 and armv7